src/commands.c

```diff
@@ -498,6 +498,8 @@ split_quoted (const gchar *src, const gboolean unquote)
         } else if (isspace (*p) && !ctx_double_quote && !ctx_single_quote) {
             /* Argument separator. */
             /* FIXME: Is "a  b" three arguments? */
+            for (; isspace(*(p + 1)); ++p);
```

Comment:
it's going to look a bit derpy anyway because I need it to stop AT the last space and not one past it.